contracts/VotingEscrow.vy

```diff
@@ -433,7 +440,6 @@ def deposit_for(_addr: address, _value: uint256):
     @param _addr User's wallet address
     @param _value Amount to add to user's lock
     """
-    assert(self.unlocked == False) # dev: no more lock
```

Comment:
Question:

I see some LoC deleted with the code `assert(self.unlocked == False)`

Shouldn't use `self.migrated` instead?

Reply:
`_deposit_for` is doing the check for all deposits, lock extension...